src/main/java/frc/robot/RobotContainer.java

```diff
@@ -55,5 +61,25 @@ private void configureButtonBindings() {
             
     }
 
+    private void autoConfig() {
+        autoChooser.setDefaultOption("1- 1 Cube + Balance", "1-1_Cube_Balance");
+        autoChooser.addOption("1- 2 cube", "1-2_Cube_Balance");
+        autoChooser.addOption("3- 1 Cube + Balance", "3-1_Cube_Balance");
+        autoChooser.addOption("3- 2 Cube", "3-2_Cube");
```

Comment:
What is the meaning of these names (how do I read them)? Maybe leave a comment in the code explaining the format of the name